tests/unit/validation/test_utils.py

```diff
@@ -76,6 +76,21 @@ def test_update_input_data():
     np.testing.assert_array_equal(merged["sym_load"]["q_specified"], [4.1, 5.2, 6.1, np.nan, np.nan, 3.2])
 
 
+def test_update_input_data__without_ids():
```

Comment:
Is the `__` intentional? XD

Reply:
Yes, update_input_data is the function
And without ids is the sub case